css/style.css

```diff
@@ -0,0 +1,226 @@
+@media(min-width: 800px){
+    body{
+        background: #ECEEF1;
+        font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+        font-weight: normal;
+    }
+}
+
+fieldset {
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+    font-size: 13px;
+    
+}
+
+p{
+    margin: 0 0 0 0;
+    font-size: 13px;
+    line-height: 1;
+}
+
+/*Header Section CSS*/
+
+.container{
+    margin: 0 0 0 0;
+}
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.navbar-text {
+    align-self: center;
+    text-align: right;
+    margin-top: 10px;
+}
+
+.navbar-btn {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+}
```

Comment:
- [ ] **[MANDATORY]** For good practices in CSS, do not repeat the same rules in different classes, see this one is also repeated:
```
.main-section {
    display: flex;
    justify-content: center;
    align-items: center;
}
```